src/main/java/org/libtorrent4j/AnnounceFlags.java

```diff
@@ -0,0 +1,27 @@
+package org.libtorrent4j;
+
+/**
+ * @author aldenml
+ */
+public final class AnnounceFlags {
+
+    private AnnounceFlags() {
+    }
+
+    /**
+     * Announce to DHT as a seed.
+     */
+    public static final byte SEED = 1; // 1 << 0
```

Comment:
Codacy found an issue: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/aldenml/libtorrent4j/pullRequest?prid=5839370)